projects/demo/src/app/datagrid/sorting/sorting.html

```diff
@@ -74,15 +74,21 @@ <h2>Custom sort</h2>
   <b>{{sortOrder === 1 ? " ascendingly" : sortOrder === -1 ? " descendingly" : ""}}</b>.
 </p>
 <p>
-  <button class="btn btn-secondary" (click)="sortOrder = 1" [disabled]="sortOrder === 1">Sort ascendingly</button>
-  <button class="btn btn-secondary" (click)="sortOrder = -1" [disabled]="sortOrder === -1">Sort descendingly</button>
-  <button class="btn btn-secondary" (click)="sortOrder = 0" [disabled]="sortOrder === 0">Clear sort</button>
+  <button class="btn btn-secondary" (click)="setSortOrder(1)" [disabled]="sortOrder === 1">Sort ascendingly</button>
+  <button class="btn btn-secondary" (click)="setSortOrder(-1)" [disabled]="sortOrder === -1">Sort descendingly</button>
+  <button class="btn btn-secondary" (click)="setSortOrder(0)" [disabled]="sortOrder === 0">Clear sort</button>
```

Comment:
Why were these changed from direct assignment? The `setSortOrder` just sets the `sortOrder` property, so it is unnecessary and should be removed.

Reply:
corrected in commit 3684b86